### PR TITLE
[fastlane] Refactor fastlane runner to make it easy to access from docs

### PR DESCRIPTION
**Pull request by KrauseFx**
Something that will help us with docs

### CODE REVIEW

File: fastlane/lib/fastlane/runner.rb [hunk 1]
```diff
@@ -82,15 +82,28 @@ def available_lanes(filter_platform = nil)
...
+    def method_string_from_method_sym(method_sym)
       method_str = method_sym.to_s
       method_str.delete!('?') # as a `?` could be at the end of the method name
+      return method_str
+    end
+
+    def class_reference_from_action_name(method_sym)
+      method_str = method_string_from_method_sym(method_sym)
 
       # First, check if there is a predefined method in the actions folder
```
Comment (mfurtak) [inline fastlane/lib/fastlane/runner.rb]: Comment moved to line 104, so should be deleted here?

File: fastlane/lib/fastlane/runner.rb [hunk 2]
```diff
@@ -82,15 +82,28 @@ def available_lanes(filter_platform = nil)
...
     end
 
-    # This is being called from `method_missing` from the Fastfile
-    # It's also used when an action is called from another action
-    def trigger_action_by_name(method_sym, custom_dir, *arguments)
+    def method_string_from_method_sym(method_sym)
       method_str = method_sym.to_s
       method_str.delete!('?') # as a `?` could be at the end of the method name
+      return method_str
+    end
```
Comment (mfurtak) [inline fastlane/lib/fastlane/runner.rb]: Could this be simplified to:

```ruby
def method_string_from_method_sym(method_sym)
  # as a `?` could be at the end of the method name
  method_sym.to_s.delete('?')
end
```

**Comment by mfurtak** [comment]
Seems good - can you be more specific about why it helps?

**Comment by KrauseFx** [comment]
Thanks for the review @mfurtak, I'll update the PR. 

Why do we need this?

We use it when verifying that the code samples in the docs are still working as expected, the method is being called here: https://github.com/fastlane/docs/pull/56/files#diff-b9ec8760f6bfcc35a81f1164b8cf0f5cR40

This is needed to fetch a reference to the action based on the a symbol, to fetch the available parameters

**Comment by KrauseFx** [comment]
Great feedback, thanks @mfurtak, I noticed that I also didn't add any comments to the code base, so I added it, and simplified the code 👍  Let me know if that looks good to you

**Comment by mfurtak** [comment]
Great comments! LGTM 👍

**Pull request merged**